scene/resources/fog_material.cpp

```diff
@@ -159,7 +159,7 @@ uniform sampler3D density_texture: hint_default_white;
 void fog() {
     DENSITY = density * clamp(exp2(-height_falloff * (WORLD_POSITION.y - OBJECT_POSITION.y)), 0.0, 1.0);
     DENSITY *= texture(density_texture, UVW).r;
-    DENSITY *= pow(clamp(-SDF / min(min(EXTENTS.x, EXTENTS.y), EXTENTS.z), 0.0, 1.0), edge_fade);
+    DENSITY *= pow(clamp(-0.5 * SDF / min(min(SIZE.x, SIZE.y), SIZE.z), 0.0, 1.0), edge_fade);
```

Comment:
Shouldn't this be `2.0` if `SIZE` is on the denominator?

Reply:
No that won't work. Actually @clayjohn had the same idea first:
![image](https://user-images.githubusercontent.com/66004280/215819581-bc229eb9-0e7d-428e-a56f-834b419eeefa.png)

Reply:
Well, the original code was:
```
-SDF / EXTENTS
```
Now we have `EXTENTS = 0.5 * SIZE`, so:
```
-SDF / (0.5 * SIZE)
```

Multiplying both sides with 2, it gives:
```
-2.0 * SDF / SIZE
```

Reply:
Ahh good catch, I see the problem. Yes, this is right. Damn braces. 😄